packages/interpretations/src/components/DetailsPanel/Description.js

```diff
@@ -24,15 +24,15 @@ export const Description = ({ displayDescription, isToggled, onToggleDescription
         <p style={{fontStyle: 'italic'}}>{description}</p>;
 
     return (
-        <Fragment>
+        <div style={{display: 'flex', flexDirection: 'column'}}>
```

Comment:
use a class instead of inline `style`?

Reply:
I had decided to keep the surface area of changes low since this component is legacy and will be replaced eventually (don't know when though). Inline styles are already used in many places in the Interpretations components.